### PR TITLE
ADD yaml format to the service container configuration.

### PR DESCRIPTION
**Pull request by chigix**
Signed-off-by: Richard Lea <chigix@zoho.com>

### CODE REVIEW

File: Resources/doc/reference/usage.rst [hunk 1]
```diff
@@ -113,6 +113,15 @@ message is receive or created.
         </services>
     </container>
 
+.. code-block:: yaml
```
Comment (Soullivaneuh) [inline Resources/doc/reference/usage.rst]: Could you please specify both formats as a list?

Something like this: https://sonata-project.org/bundles/admin/master/doc/reference/getting_started.html#step-3-create-an-admin-service

Related doc code: https://raw.githubusercontent.com/sonata-project/SonataAdminBundle/master/Resources/doc/reference/getting_started.rst

File: Resources/doc/reference/usage.rst [hunk 2]
```diff
@@ -113,6 +113,15 @@ message is receive or created.
...
+
+    services:
+        sonata.notification.consumer.logger:
+            class: Sonata\NotificationBundle\Consumer\LoggerConsumer
+            tags:
+                - { name: sonata.notification.consumer, type: logger }
+            arguments: ["@logger"]
```
Comment (Soullivaneuh) [inline Resources/doc/reference/usage.rst]: `arguments: [ @logger ]` instead.

**Comment by Soullivaneuh** [comment]
ping @chigix

**Comment by chigix** [comment]
Sorry for response so late, I've push an extra commit with the list format and the change for the service argument.

However, I've test there's no effect no matter with the quotation marks in the previous commit. In addition, I saw the official document from symfony also write quotation marks in the [service container section](http://symfony.com/doc/current/book/service_container.html#referencing-injecting-services).

**Comment by Soullivaneuh** [comment]
Could you please squash your commit and rebase your PR to make Travis green?

Thanks.

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by Soullivaneuh** [comment]
Thank you @chigix.

**Pull request merged**